packages/terra-action-header/src/terra-dev-site/doc/action-header/ActionHeader.1.doc.mdx

```diff
@@ -1,3 +1,16 @@
+import { Badge } from '../../../../package.json?dev-site-package';
+
+import DefaultActionHeader from '../example/DefaultActionHeader?dev-site-example';
+import BackActionHeader from '../example/BackActionHeader?dev-site-example';
+import BackCloseActionHeader from '../example/BackCloseActionHeader?dev-site-example';
+import CloseActionHeader from '../example/CloseActionHeader?dev-site-example';
+import MaximizeCloseActionHeader from '../example/MaximizeCloseActionHeader?dev-site-example';
+import MinimizeCustomButtonActionHeader from '../example/MinimizeCustomButtonActionHeader?dev-site-example';
+import BackPreviousNextCloseActionHeader from '../example/BackPreviousNextCloseActionHeader?dev-site-example';
+import LongWrappingTextHeader from '../example/LongWrappingTextHeader?dev-site-example';
+
+import ActionHeaderPropsTable from '../../../../src/ActionHeader?dev-site-props-table';
+
```

Comment:
We should be rendering the Badge at the top of these docs

Reply:
updated here https://github.com/cerner/terra-core/pull/2802/commits/5339e5c49685a2febe7d1fd150600dc7892892f5.

Reply:
@pranav300 lets add these to the Changelog and Upgrade guides as well so they read the same way as when they were rendered with Doc Template

Reply:
updated here https://github.com/cerner/terra-core/pull/2802/commits/436ab51b2819072f156c3711708febc83c3bd277.